front-end/src/components/navbar.js

```diff
@@ -109,33 +110,28 @@ const Navbar = () => {
 
         <div className="main">
           <div className="user">
-            <i className="bx bxs-user"></i>Sign In
+            <i class="bx bxs-user"></i>Sign In
```

Comment:
In react you can't use class keyword for css, you have to use "className".

Reply:
Ok